client/jetpack-connect/controller.js

```diff
@@ -31,6 +34,12 @@ import { isJetpackSite } from 'state/sites/selectors';
  */
 const debug = new Debug( 'calypso:jetpack-connect:controller' );
 const userModule = userFactory();
+const analyticsPageTitleByType = {
+	install: 'Jetpack Install',
+	personal: 'Jetpack Connect Personal',
+	premium: 'Jetpack Connect Premium',
+	pro: 'Jetpack Install Pro',
```

Comment:
With analytics it is generally best to avoid changing labels, so that we get the longest possible timeline for any particular event.